package.json

```diff
@@ -24,7 +24,8 @@
   },
   "scripts": {
     "start": "bin/polyfill-service",
-    "prepublish": "grunt updatelibrary && grunt build",
+    "prepublish": "npm run build",
+    "build": "grunt updatelibrary && grunt build",
```

Comment:
Can we make this a grunt task?  It seems like making a confusing situation more confusing to define this task in package scripts.